optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/DefaultPartitionedSearchPhase.java

```diff
@@ -184,10 +185,8 @@ private ExecutorService createThreadPoolExecutor(int partCount) {
             phaseList.add(phase);
             partPhaseIndex++;
         }
-        // TODO create PartitionSolverScope alternative to deal with 3 layer terminations
-        SolverScope<Solution_> partSolverScope = solverScope.createChildThreadSolverScope(ChildThreadType.PART_THREAD);
-        partSolverScope.setRunnableThreadSemaphore(runnablePartThreadSemaphore);
```

Comment:
nitpick: Here setRunnableThreadSemaphore() is replaced for a constructor parameter. Which is usually a good thing. 1/2